src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js

```diff
@@ -36,9 +36,12 @@ export const BasicSliderInput = (props) => {
     ...otherProps
   } = props;
 
+  const min = typeof minValue === 'number' ? minValue : 0;
+  const max = typeof maxValue === 'number' ? maxValue : 100;
+
   const getValue = (value) => {
-    if (value == null || isNaN(value)) {
-      return 0;
+    if (typeof value !== 'number' || isNaN(value)) {
```

Comment:
I think `typeof value !== 'number' ` and `isNaN(value)` do overlaps.

The first test is maybe sufficient.

Reply:
unfortunately, no because the type of `NaN` is `number` (yes, JS **IS** weird) :woman_shrugging: